src/components/alert/alert.tsx

```diff
@@ -226,7 +231,12 @@ export class Alert implements OpenCloseComponent, LocalizedComponent {
             <slot name={SLOTS.message} />
             <slot name={SLOTS.link} />
           </div>
-          {queueCount}
+          {hasActionEnd ? (
+            <div class={CSS.actionsEnd}>
```

Comment:
Per [Stencil recommendation](https://stenciljs.com/docs/templating-jsx#dealing-with-children), can you add `key` to any conditionally rendered nodes?